### PR TITLE
Input/output files

### PR DESCRIPTION
**Pull request by benjamin-owen**
File I/O for input, memory, and output files

### CODE REVIEW

File: artifacts/meminit.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+0x00000000,0xDEADBEEF,1
```
Comment (psrisank) [inline artifacts/meminit.csv:1]: I believe that the memory should be initialized with a state for every node.

Is M=0, S=1, I=3?

File: README.md [hunk 2]
```diff
@@ -21,4 +25,18 @@ Currently, a test packet is created and placed into the compute node's TX queue.
...
 
-Next steps are to add logic to handle sending to different memory nodes/compute nodes through a single switch (different ports).
\ No newline at end of file
+Next steps are to add logic to handle sending to different memory nodes/compute nodes through a single switch (different ports).
+
+## Packet input/log file format
+
+The input/log file are csv (comma separated values) files with the following columns:
+
+**Time, Packet ID, Flag, src, dst, Address, Data**
```
Comment (psrisank) [inline README.md:34]: I think we should have the input log say if its a read or write instruction. I don't see any provisioning for actually modifying the data itself in the input file.

Comment (benjamin-owen) [reply README.md:34]: Should be included in the "Flag" field.  See "packet.h" NORMAL=0, READ=1, WRITE=2, ERROR=3.

Comment (psrisank) [reply README.md:34]: oops didn't see that

**Pull request merged**